packages/backup-restore-operator/charts/templates/deployment.yaml

```
@@ -0,0 +1,53 @@
+apiVersion: apps/v1 
+kind: Deployment
+metadata:
+  name: backup-restore-operator
+  namespace: {{ .Release.Namespace }}
+  labels:
+    resources.cattle.io/operator: backup-restore
```

@deniseschannon @cbron should these new Rancher charts be updated to have the recommended labels from Helm 3 described [here](https://helm.sh/docs/chart_best_practices/labels/)? It's not a major change and could be pretty easily introduced with a helper like in `rancher-pushprox` here:

https://github.com/rancher/charts/blob/98d70cee35a19afd217633b5283e21a0dd0b8296/packages/rancher-pushprox/charts/templates/_helpers.tpl#L11-L15

This would then be changed to:
```suggestion
  labels: {{ include "backupRestore.labels" . | nindent 4 }}
```

---

Probably but not a blocker for being merged.